src/dataflow/Cargo.toml

```diff
@@ -42,6 +42,7 @@ ore = { path = "../ore", features = ["task"]  }
 pdqselect = "0.1.1"
 persist = { path = "../persist" }
 persist-types = { path = "../persist-types" }
+pgwire = { path = "../pgwire" }
```

Comment:
unsure if this is something we want to do. feels a bit icky, but since we are trying to populate the data from postgres in `dataflow` already, we do need to be able to parse it out.

Reply:
Oof, yeah, would be great to avoid this! Is it hard to move the `CopyTextFormatParser` into a new `pgcopy` crate?

Reply:
👍 will take a look at this